pkg/kubelet/cm/memorymanager/memory_manager.go

```diff
@@ -331,8 +331,7 @@ func getTotalMemoryTypeReserved(machineInfo *cadvisorapi.MachineInfo, reservedMe
 
 	for _, reservation := range reservedMemory {
 		if !numaNodes[int(reservation.NumaNode)] {
-			klog.Warningf("The NUMA node %d specified under --reserved-memory does not exist on the machine", reservation.NumaNode)
-			continue
+			return nil, fmt.Errorf("[memory manager] NUMA node %d specified under reserved memory does not exist on the machine", reservation.NumaNode)
```

Comment:
We don't generally include things like `[memory manager]` in error text, the style would be to say `the reserved memory configuration references a NUMA node %d that does not exist on this machine`.  The message is unambiguous

Reply:
done